pkg/api/types.go

```diff
@@ -385,6 +388,9 @@ const (
 	// BetaStorageClassAnnotation represents the beta/previous StorageClass annotation.
 	// It's currently still used and will be held for backwards compatibility
 	BetaStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
+	// AlphaStorageTopologyAnnotation is set in the StorageClass and gives the scheduler
+	// a hint that the volumes in this StorageClass have a topology constraint
+	AlphaStorageTopologyAnnotation = "storage.alpha.kubernetes.io/topology-key"
```

Comment:
I'm not sure about the behavior of just specifying the key without an operator or value.  For pod affinity, they actually have two fields, a label selector and a topologyKey.  The topologyKey field means to match the labels between two objects (in this case, PV and Node).  Whereas the selector is applying the filters to a single object.

I also spoke with @lavalamp about using labels to express the topology constraints.  Would we ever want to express in the PV more than a single label/value?  Like, this PV can be accessed on node1,2, and 3?  Or define a new single label "mylabel=foo", and put that same label/value on node1,2,3?